src/catchup/simulation/ApplyTransactionsWork.h

```diff
@@ -26,24 +26,37 @@ class ApplyTransactionsWork : public BasicWork
     std::vector<TransactionResultPair>::const_iterator mResultIter;
 
     uint32_t const mMaxOperations;
+    uint32_t const mMultiplier;
```

Comment:
How is `mSimulateOnly` different from `mMultiplier == 1`? If they're the same, I'd prefer to use only `mMultiplier = 1` as the canonical "simulate-only".